packages/content-management/table_list_view_table/src/components/table_sort_select.tsx

```diff
@@ -165,8 +192,8 @@ export function TableSortSelect({ tableSort, hasUpdatedAtMetadata, onChange }: P
       <>
         <EuiText css={panelHeaderCSS}>{i18nText.headerSort}</EuiText>
         <EuiSelectable<SortItem>
-          singleSelection
-          aria-label="some aria label"
+          singleSelection={'always'}
+          aria-label="Sorting options"
```

Comment:
Good catch! The text should probably be translatable though:
```suggestion
          aria-label={i18n.translate(
            'contentManagement.tableList.listing.tableSortSelect.sortingOptionsAriaLabel',
            { defaultMessage: 'Sorting options' }
          )}
```